client/src/components/common/applicant-info-1.vue

```diff
@@ -19,7 +19,7 @@
                             height="50"
                             hide-details="auto"
                             id="lastname"
-                            name="lastname"
+                            :name="Math.random()"
```

Comment:
I'm testing this locally and it's still allowing me to "fill the form" using values saved in Chrome.

Can you look into creating a Vue directive for this?
https://vuejs.org/v2/guide/custom-directive.html